src/Scylla/main.ts

```diff
@@ -0,0 +1,367 @@
+    }
+
+    async getSearchFilters(): Promise<SearchFilter[]> {
+        const filters: SearchFilter[] = [];
+
+        const searchTags = await this.getGenreTags();
+        for (const tags of searchTags) {
+            filters.push({
+                type: "multiselect",
+                options: tags.tags.map((x) => ({ id: x.id, value: x.title })),
+                id: tags.id,
+                allowExclusion: true,
+                title: tags.title,
+                value: {},
+                allowEmptySelection: true,
+                maximum: undefined,
+            });
+        }
+
+        // Register Sort By Filter
+        filters.push({
+            id: "sortBy",
+            type: "dropdown",
+            options: [
+                { id: "Random", value: "Random" },
+                { id: "New", value: "New" },
+                { id: "Updated", value: "Updated" },
+                { id: "Views", value: "Views" },
+            ],
+            value: "Views",
+            title: "Sort By Filter",
+        });
+
+        return filters;
+    }
+
+    private async getFilteredSectionItems(
+        filter: string,
+        metadata: Metadata | undefined,
+    ): Promise<PagedResults<DiscoverSectionItem>> {
+        if (metadata?.completed) return EndOfPageResults;
+
+        const page: number = metadata?.page ?? 1;
+
+        const request: Request = {
+            url: new PBURL(SCYLLA_DOMAIN)
+                .addPathComponent("manga")
+                .setQueryItem("page", String(page))
+                .setQueryItem("filter", filter)
+                .toString(),
+            method: "GET",
+        };
+        const $ = await this.fetchCheerio(request);
+        const manga = parseViewMore($);
+        metadata = !isLastPage($) ? { page: page + 1 } : undefined;
+
+        return {
+            items: manga,
+            metadata: metadata,
+        };
+    }
+
+    async getGenreSectionItems(): Promise<PagedResults<DiscoverSectionItem>> {
+        const genres = (await this.getGenreTags())[0];
+
+        return {
+            items: genres.tags.map((genre) => ({
+                type: "genresCarouselItem",
+                searchQuery: {
+                    title: "",
+                    filters: [
+                        { id: "genres", value: { [genre.id]: "included" } },
+                    ],
+                },
+                name: genre.title,
+                metadata: undefined,
+            })),
+            metadata: undefined,
+        };
+    }
+
+    // may need to check for cf headers if challenges appear in future
+    checkCloudflareStatus(status: number): void {
+        if (status === 503) {
+            throw new CloudflareError({ url: SCYLLA_DOMAIN, method: "GET" });
+        }
+        if (status === 403) {
+            throw new Error(
+                "Server returned 403 Forbidden. Logins not yet supported. Let me know on Discord if you can view this title when logged in on the website.",
```

Comment:
Users only have the contact information you list in the developer section of the `pbconfig.ts` file, you probably want to be more descriptive with your contact information. Asking them to make a support post in the Inkdex extension repository on GitHub or Discord server would be better imo (links are not interactive or copyable in Paperback).
```suggestion
                "Status code: 403 Forbidden. Login is not yet supported. Make a support issue/post in the Inkdex extension repository on their GitHub Organization or Discord server if you can view this title when logged in on the website.",
```

Reply:
I at first believed it was for users who could login only, however I joined their discord to ask about it and they just send 403 responses for title pages that got taken down via DMCA, and for some reason leave the titles listed. I updated the comment to reflect this.